sktime/forecasting/model_selection/_split.py

```diff
@@ -840,8 +804,7 @@ def __init__(
     def _initial_window(self):
         if hasattr(self, "initial_window"):
             return self.initial_window
-        else:
-            return None
+        return None
```

Comment:
why?

Reply:
same as in the other comment (https://github.com/sktime/sktime/pull/4108#discussion_r1070364781)